github/util.py

```diff
@@ -221,3 +224,95 @@ def retrieve_email(username: str):
         )
     )
 
+def _create_team(
+    github: GitHub,
+    organization_name: str,
+    team_name: str
+):
+    # passed GitHub object must have org. admin authorization to create a team
+    organization = github.organization(organization_name)
+    team = _retrieve_team_by_name(organization, team_name)
+    if team:
+        util.verbose("Team {name} already exists".format(name=team_name))
+        return
+
+    try:
+        organization.create_team(name=team_name)
+        util.info("Team {name} created".format(name=team_name))
+    except ForbiddenError as err:
+        util.fail("{err} Cannot create team {name} in org {org} due to missing privileges".format(
+            err=err,
+            name=team_name,
+            org=organization_name
+        ))
+
+def _add_user_to_team(
+    github: GitHub,
+    organization_name: str,
+    team_name: str,
+    user_name: str
+):
+    # passed GitHub object must have org. admin authorization to add a user to a team
+    organization = github.organization(organization_name)
+    team = _retrieve_team_by_name(organization, team_name)
+    if not team:
+        util.fail("Team {name} does not exist".format(name=team_name))
+
+    if team.is_member(user_name):
+        util.verbose("{username} is already assigned to team {teamname}".format(
+            username=user_name,
+            teamname=team_name
+        ))
+        return
+
+    if team.add_member(username=user_name):
+        util.info("Added {username} to team {teamname}".format(
+            username=user_name,
+            teamname=team_name
+        ))
+    else:
+        util.fail("Could not add {username} to team {teamname}. Please check for missing privileges".format(
+            username=user_name,
+            teamname=team_name
+        ))
+
+
+class RepoPermission(Enum):
+    pull = "pull"
+    push = "push"
+    admin = "admin"
+
+
+def _add_all_repos_to_team(
+    github: GitHub,
+    organization_name: str,
+    team_name: str,
+    permission: RepoPermission=RepoPermission.admin
+):
```

Comment:
would be nice to document the function's semantics; for the sake of clear(er) structuring, it could be considered to accept an iterable of repositories in this function and retrieve the organisation's repositories outside of this function. admittedly, retrieving an organisation's repositories is a trivial one-liner.